src/openepd/model/specs/generated/enums.py

```diff
@@ -0,0 +1,2426 @@
+
+      * `csa.S-1` - Concrete subjected to very severe sulphate exposures.
+                    Exposed to >10000 ppm of SO4 in water or >2% SO4 in soil
+      * `csa.S-2` - Concrete subjected to severe sulphate exposure.
+                    Exposed to <10000 ppm of SO4 in water and <2% SO4 in soil
+      * `csa.S-3` - Concrete subjected to moderate sulphate exposure.
+                    Exposed to <1500 ppm of SO4 in water and <0.2% SO4 in soil
+    """
+
+    C_XL = "csa.C-XL"
+    C_1 = "csa.C-1"
+    C_2 = "csa.C-2"
+    C_3 = "csa.C-3"
+    C_4 = "csa.C-4"
+    F_1 = "csa.F-1"
+    F2 = "csa.F-2"
+    N = "csa.N"
+    S_1 = "csa.S-1"
+    S_2 = "csa.S-2"
+    S_3 = "csa.S-3"
+    A_1 = "csa.A-1"
+    A_2 = "csa.A-2"
+    A_3 = "csa.A-3"
+    A_4 = "csa.A-4"
+
+
+class EnExposureClass(StrEnum):
+    """
+    EN 206 Class (Europe).
+
+    European Standard concrete exposure classes.
+
+      * `en206.X0` - No risk of corrosion or attack.
+
+      Corrosion induced by carbonation.
+
+      * `en206.XC1` - Dry or permanently wet.
+      * `en206.XC2` - Wet, rarely dry.
+      * `en206.XC3` - Moderate humidity.
+      * `en206.XC4` - Cyclic wet and dry.
+
+      Corrosion induced by chlorides from sea water.
+
+      * `en206.XS1` - Exposed to airborne salt but not in direct contact with sea water.
+      * `en206.XS2` - Permanently submerged.
+      * `en206.XS3` - Tidal, splash and spray zones.
+
+      Corrosion induced by chlorides other than from sea water.
+
+      * `en206.XD1` - Moderate humidity.
+      * `en206.XD2` - Wet, rarely dry.
+      * `en206.XD3` - Cyclic wet and dry.
+
+      Freeze/thaw attack with or without de-icing agents.
+
+      * `en206.XF1` - Moderate water saturation, without deicing agent.
+      * `en206.XF2` - Moderate water saturation, with deicing agent.
+      * `en206.XF3` - High water saturation, without de-icing agent.
+      * `en206.XF4` - High water saturation, with de-icing agent or sea water.
+
+      Chemical attack.
+
+      * `en206.XA1` - Slightly aggressive chemical environment.
+      * `en206.XA2` - Moderately aggressive chemical environment.
+      * `en206.XA3` - Highly aggressive chemical environment.
+
+    """
+
+    en206_X0 = "en206.X0"
+    en206_XC1 = "en206.XC1"
+    en206_XC2 = "en206.XC2"
+    en206_XC3 = "en206.XC3"
+    en206_XC4 = "en206.XC4"
+    en206_XS1 = "en206.XS1"
+    en206_XS2 = "en206.XS2"
+    en206_XS3 = "en206.XS3"
+    en206_XD1 = "en206.XD1"
+    en206_XD2 = "en206.XD2"
+    en206_XD3 = "en206.XD3"
+    en206_XF1 = "en206.XF1"
+    en206_XF2 = "en206.XF2"
+    en206_XF3 = "en206.XF3"
+    en206_XF4 = "en206.XF4"
+    en206_XA1 = "en206.XA1"
+    en206_XA2 = "en206.XA2"
+    en206_XA3 = "en206.XA3"
+
+
+class AggregateWeightClassification(StrEnum):
+    """Aggregate weight classification enum."""
+
+    LIGHTWEIGHT = "Lightweight"
+    """
+    Also called "low-density aggregate". Aggregate with bulk density less than 1120 kg/m3 such as: pumice, scoria,
```

Comment:
Did you intentionally deviate from the convention of providing a per-class docstring, as introduced earlier?

Reply:
This is a generated part of code - the documentation (and docstrings) improvement will be handled in a several batches as a part of https://app.asana.com/0/0/1206887306120203/f

Reply:
I think we will need to move these to the class string, because a) it is picked up by documentation tools and IDE and b) rendered as docs in stoplight.